Changelog

```diff
@@ -1,3 +1,45 @@
+Unreleased - sledgehammer999 <sledgehammer999@qbittorrent.org> - v5.0.0
+    - FEATURE: Drop support for Qt5, qmake, autotools, Windows < 10, Windows 32-bit
+    - FEATURE: Support creating .torrent with larger piece size (Chocobo1) 4d2015cfeda95be0fd9f6f9628ab77d48c741d17
+    - FEATURE: Improve tracker entries handling (glassez) 7cd2445a49e879255dca5015592ea94ccd9d0d12
+    - FEATURE: Add separate filter item for tracker errors (glassez) 3ca093941108c27ee24ad984877baf0ee17d961e
+    - FEATURE: Allow to remove tracker from tracker filter widget menu (glassez) 30189ae943b9a85d39484f6cdc64e2200518654f
+    - FEATURE: Implement `Reannounce In` column (Hanabishi) c394868f87e71d8ab614c6ae5463c712acffd172
+    - FEATURE: Expose `DHT bootstrap nodes` setting (Chocobo1) dcba9eda009fcb74e8aebbee4a4c5df7248abf06
+    - FEATURE: Improve GUI behavior when adding multiple torrents (glassez) 70b438e6d9249bacf8589ba50d16bbfd617fd199
+    - FEATURE: Add support for [Mark-of-the-Web](https://redcanary.com/threat-detection-report/techniques/mark-of-the-web-bypass/) (Chocobo1) 34f7b75f12390454ed831037ff34531b90307c55
+    - FEATURE: Keep unwanted files in separate folder (configurable) (glassez) e216a9cf53ae1203d148a013ab5d71d6672fe97a
```

Comment:
I'm afraid `(configurable)` will give the wrong impression that it's location can be changed.

Reply:
I'll drop `(configurable)` and let them discover the setting on their own.

Reply:
Maybe `Allow to keep unwanted files in separate folder`?